modules/domUtils.js

```diff
@@ -0,0 +1,27 @@
+// import { showContent } from './domUtils.js';
+
+/* eslint-disable */
+
+function updateActiveNavLink(selectedLink) {
```

Comment:
- [ ] Save here too as pointed out earlier :point_up: Kindly change this function to **arrow** function syntax.